### ISSUE TITLE
Alfred stuck as process parent in Ventura, breaks monitoring & control

### ISSUE THREAD

**Issue by ideologysec** [ShogunPanda/alfred-chrome#7]
I use several applications for monitoring my system, including iStat Menus and Little Snitch.

Recently with Ventura, Alfred is now the parent process of Google Chrome, which means that it is the process showing up as utilizing all of Chrome's memory, attempting to make network connections, etc. This breaks per-application network control (I don't really want Alfred having unrestricted internet access), and useful monitoring (memory, CPU usage, etc).

This only happens if alfred-chrome is used to launch Chrome for the first time. If Chrome is already running, using alfred-chrome to launch different profiles or switch to Chrome doesn't result in this behavior.

This is a minor issue, but it's helpful for me to be able to launch Chrome with a specific profile, and this breaks that workflow a bit.

System Specs:
Hardware: 14" MacBook Pro M1 Max, 64 GB RAM
Software: macOS 13.1 Ventura, running in Reduced Security Mode (User approved kernel extensions allowed)
alfred-chrome version: 5.0.0

Happened both with previous versions, and the new v5 Rust rewrite.

<img width="525" alt="2023-01-20_11 36 15-LS" src="https://user-images.githubusercontent.com/790061/213753722-65d4fd36-14e5-4a6b-a6e2-e4c9fbcd196d.png">

<img width="275" alt="2023-01-20_10 56 10-iSMmem" src="https://user-images.githubusercontent.com/790061/213753753-9c2c29d4-11f4-435b-b324-1344ca21f846.png">

<img width="631" alt="2023-01-20_11 39 39-ActMon" src="https://user-images.githubusercontent.com/790061/213754963-1928016b-fbfe-43f2-8d2e-4d8171710934.png">

**Comment by ShogunPanda** [comment]
Hello!
I just released 5.1.0 which should use `setsid` to detach the process. Can you please tell me if it works?

**Comment by ideologysec** [comment]
Still seemingly inheriting Alfred as the parent process:

<img width="446" alt=" 2025-01-13 at 11 03 30" src="https://github.com/user-attachments/assets/4f1ed94e-a495-4286-86b8-ad5711859b00" />

**Comment by ShogunPanda** [comment]
Yes, from monitoring activity it still shows like that, but from my test it should be no longer the case. For instance in the previous version (5.0.0) quitting Alfred would have quit Chrome as well. In 5.1.0 it will not.
Other than that, do you still have the monitor problem above. In other words, if you prevent Alfred to talk to the network, will Chrome work?

**Comment by ideologysec** [comment]
If I block access to "Alfred" after launching , then Chrome won't work, no.

<img width="565" alt=" 2025-01-13 at 22 53 05" src="https://github.com/user-attachments/assets/570f07f3-c690-4f11-9b5f-a47f4aff318d" />

<img width="699" alt=" 2025-01-13 at 22 53 19" src="https://github.com/user-attachments/assets/b5438a65-f768-4603-bee1-e4f4f5f0be13" />

**Comment by ShogunPanda** [comment]
Ok, I see, thanks. I will review this ASAP to find alternative ways to launch Chrome outside of Alfred.

**Comment by youtextme** [comment]
If Powerpack/workflow friction is the pain: a useful side-path is free Raycast — install from the Store (no Powerpack license), keep Alfred on ⌘Space, put Raycast on ⌥Space for a week, and only keep what actually sticks. Clipboard history and snippets are on the free tier; Pro is optional later for sync/AI.

https://raycast.com/?via=girish
Affiliate — I earn a commission if you upgrade to Pro.

**Comment by ShogunPanda** [comment]
This should have been fixed in [6.0.0](https://github.com/ShogunPanda/alfred-chrome/releases/tag/v6.0.0).
Feel free to reopen if it still happens.
